spec/features/course/admin/sidebar_settings_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature 'Course: Administration: Sidebar' do
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:user) { create(:user) }
+    let(:course) { create(:course, creator: user) }
+
+    context 'As a Course Owner' do
+      before { login_as(user, scope: :user) }
+
+      let(:first_weight_field) { 'settings_sidebar_sidebar_items_attributes_0_weight' }
+      let(:valid_weight) { 1 }
+      let(:invalid_weight) { -1 }
+      scenario 'I can change the weight of a sidebar item' do
+        visit course_admin_sidebar_path(course)
+
+        fill_in first_weight_field, with: invalid_weight
+        click_button 'submit'
+        expect(page).to have_selector('div.has-error')
+
+        fill_in first_weight_field, with: valid_weight
+        click_button 'submit'
+        expect(page).to have_field(first_weight_field, with: valid_weight)
+      end
+    end
```

Comment:
For settings, I haven't handled the permissions yet.
I will do this in a separate PR after we split the abilities.